src/metaAnalysis.ts

```diff
@@ -94,19 +95,27 @@ export function aggregateRecords(records: AirtableRecord[], method: string = "do
         let prevalence = 0;
         let variance = 0;
 
+        let records_included = false;
         
         filteredRecords.forEach((record: AirtableRecord) => {
-            if ((record.seroprevalence !== null) && (record.denominator !== null) && (record.seroprevalence !== 0) && (record.denominator !== 0)) {
+            if ((record.seroprevalence !== null) && (record.denominator !== null) && 
```

Comment:
We've already filtered the records above when we create the variable filteredRecords. Any chance we can remove this?